docs(kong-user) switching to a non-kong user (#2505)
Kong/docs.konghq.com: app/2.2.x/kong-user.md

* [RedHat](/install/redhat)
* [Ubuntu](/install/ubuntu)

## Run Kong Gateway as the built-in kong user

> When {{site.ce_product_name}} is installed with a package management system such as `APT` and `YUM`, a default `kong` user and a default `kong` group are created. All the files installed by the package are owned by the `kong` user and group.

Reviewer: ```suggestion
When {{site.ce_product_name}} is installed with a package management system such as `APT` or `YUM`, a default `kong` user and a default `kong` group are created. All the files installed by the package are owned by the `kong` user and group.
```

## Run Kong Gateway as a non-root user

> 1. Switch to the `kong` user:

Reviewer: ```suggestion
1. Switch to the built-in `kong` user:
```

    ```sh
    $ su kong

>     ```

Reviewer: I think there needs to be a line here to introduce the possibility of using a custom user, for example:

You can also switch to a custom non-root user. This will require an extra setup step.

    ```sh
    kong start
    ```

> If running Kong as a non-root user other than the built-in `kong` user and group is desired, consider adding the user to the “kong” group with something like:

Reviewer: ```suggestion
If running Kong as a custom non-root user is preferred, add the user to the `kong` group with a command similar to the following (the command may differ depending on your OS):
```
I might be wrong, but it seems like you wouldn't want to say "something other than the built-in `kong` user and group" if you're still telling them to use the `kong` group.

Reviewer: After looking at this flow again - I suggest moving this up as an optional step 2, because they'd want to do this before starting Kong. Eg:

2. (Optional) If running Kong ...etc

So then the task becomes:
1. Switch to user
2. Optionally assign a group
3. Start Kong

    ```sh
    kong start
    ```

## Run Kong Gateway as a custom non-root user

> It is also possible running Kong as a custom non-root user. Since all the files installed by the {{site.ce_product_name}} package are owned by the `kong` group, a user that belongs to that group should be permitted to perform the same operations as the `kong` user.

Reviewer: ```suggestion
It is also possible to run Kong as a custom non-root user. Since all the files installed by the {{site.ce_product_name}} package are owned by the `kong` group, a user that belongs to that group should be permitted to perform the same operations as the `kong` user.
```
